### PR TITLE
Padding and table styling(PROJQUAY-4351)

### PR DESCRIPTION
**Pull request by Sunandadadi**
Spacing and Tabs styling on Repos and Tags Page:
Screen Shot 2022-09-07 at 12 39 30 PM
Screen Shot 2022-09-07 at 12 39 43 PM
Removed bottom shadow on Orgs Page:
Screen Shot 2022-09-07 at 12 41 58 PM

### CODE REVIEW

**Comment by HammerMeetNail** [comment]
LGTM, do you want to include the bottom pagination in this too?

**Comment by Sunandadadi** [comment]
> LGTM, do you want to include the bottom pagination in this too?

I was thinking of doing that in a different PR.

**Review by HammerMeetNail — APPROVED** [review]

File: src/App.css [hunk 1]
```diff
@@ -29,3 +29,16 @@ html, body, #root,  .App {
     pointer-events: none;
     color: black;
 }
+
+.no-padding-bottom {
+    padding-bottom: 0 !important;
```
Comment (syed) [inline src/App.css:34]: can we remove the `!important` if that is not necessary?

Comment (Sunandadadi) [reply src/App.css:35]: It is not working without adding `!important`.

**Pull request merged**